webapp/channels/src/components/admin_console/schema_admin_settings.jsx

```diff
@@ -571,6 +577,60 @@ export default class SchemaAdminSettings extends React.PureComponent {
         );
     };
 
+    buildRolesSetting = (setting) => {
+        const {roles} = this.props;
+
+        const values = Object.keys(roles).map((r) => {
+            const text = roles[r].display_name ? (
+                <FormattedMessage
+                    id={roles[r].display_name}
+                    defaultMessage={roles[r].name}
```

Comment:
No idea how this is working out, but shouldn't this be the other way around? the defaultMessage being the display name and the id being the role name?

Reply:
Swapped